### PR TITLE
Feature/moloch pack 16022023

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

fixes:
https://github.com/HausDAO/monorepo/issues/207

fixes:
https://github.com/HausDAO/monorepo/issues/228

fixes:
https://github.com/HausDAO/monorepo/issues/229

## Changes

- new macros
- uses useProfile
- adds member display

## Packages Added

Brief list of any packages added, and if folks need to rerun `yarn install` to run locally

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for _any impacted package_ and check for any errors before the PR)

### CODE REVIEW

File: libs/moloch-v3-macro-ui/src/components/DaoSettings/MetadataSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import { Link as RouterLink } from 'react-router-dom';
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/DaoSettings/MetadataSettings.tsx:1]: cleanup

File: libs/moloch-v3-macro-ui/src/components/MemberDisplay/MemberDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { useCallback, useEffect, useState } from 'react';
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/MemberDisplay/MemberDisplay.tsx:1]: cleanup

File: libs/moloch-v3-macro-ui/src/components/MemberDisplay/MemberDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import { AccountProfile } from '@daohaus/utils';
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/MemberDisplay/MemberDisplay.tsx:2]: cleanup

File: libs/moloch-v3-macro-ui/src/components/ProposalHistory/ProposalHistoryCard.tsx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { MouseEvent, useState } from 'react';
 import { useParams } from 'react-router-dom';
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/ProposalHistory/ProposalHistoryCard.tsx:2]: cleanup useParams & AddressDisplay

File: libs/moloch-v3-macro-ui/src/components/ProposalHistory/VoteList.tsx [hunk 5]
```diff
@@ -9,6 +9,8 @@ import { AddressDisplay, DataMd, ParMd, widthQuery } from '@daohaus/ui';
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/ProposalHistory/VoteList.tsx:9]: cleanup AddressDisplay

**Review by santteegt — APPROVED** [review]

File: libs/moloch-v3-macro-ui/src/components/DaoSettings/GovernanceSettings.tsx [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+        {includeLinks && (
+          <ButtonRouterLink
+            color="secondary"
+            to={`/molochv3/${daoChain}/${
+              dao.id
+            }/new-proposal?formLego=TOKEN_SETTINGS&defaultValues=${JSON.stringify(
+              defaultValues
+            )}`}
+          >
+            <Button color="secondary">Update Token Settings</Button>
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/DaoSettings/GovernanceSettings.tsx:142]: this is throwing the following warning:

image

File: libs/moloch-v3-macro-ui/src/components/MemberDisplay/MemberDisplay.tsx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+          }
+        }
+      >
+        <MemberCardItem asChild>
+          {includeLinks && (
+            <StyledRouterLink to={customProfileURI ? customProfileURI : '/'}>
+              View Profile
+            </StyledRouterLink>
+          )}
+        </MemberCardItem>
```
Comment (0xRowdy) [inline libs/moloch-v3-macro-ui/src/components/MemberDisplay/MemberDisplay.tsx:48]: Bad things could happen if try and render the MemberCardItem asChild if there is no child. Just move the check up
```suggestion
        {includeLinks && (
          <MemberCardItem asChild>
            <StyledRouterLink to={customProfileURI ? customProfileURI : '/'}>
              View Profile
            </StyledRouterLink>
          </MemberCardItem>
        )}
```

**Review by 0xRowdy — APPROVED** [review]
Looks good. I left a few comments

**Pull request merged**